### ISSUE TITLE
Remove the term "outcome" from WCAG 3.0

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag3#496]
The latest editors draft still has a whole bunch of uses of the world "outcome", even though the group seems to have moved away from that term in favor of "requirement" (a good choice IMO). Having both terms into the draft can be confusing.

It should be.a simple search/replace to get rid of these. Here are some example:

> To meet the varying needs of this audience, several layers of guidance will be provided including guidelines written as **outcome** statements, requirements that can be tested, assertions, a rich collection of methods, resource links, and code samples.

> guidelineDeveloping
> high-level, plain-language **outcome** statements used to organize requirements

> Guidelines provide a high-level, plain-language **outcome** statements for managers, policy makers, individuals who are new to accessibility, and other individuals who need to understand the concepts but not dive into the technical details. They provide an easy-to-understand way of organizing and presenting the requirements so that non-experts can learn about and understand the concepts.
